framework/src/ics/InitialConditionWarehouse.C-e

```diff
@@ -0,0 +1,104 @@
+//* This file is part of the MOOSE framework
```

Comment:
remove this from the PR, too

Reply:
will do. Something was wrong with my clang-format and its interaction with Mac.